README.rst

```diff
@@ -190,9 +190,14 @@ Here is an end to end example of how to use a SageMaker Estimator:
     response = mxnet_predictor.predict(data)
 
     # Tears down the SageMaker endpoint
-    mxnet_estimator.delete_endpoint()
+    mxnet_predictor.delete_endpoint()
 
 
+The above example will delete endpoint and endpoint configuration at the same time. If you want to keep endpoint configuration, you can do the following:
+
+.. code:: python
+    mxnet_predictor.delete_endpoint(delete_endpoint_config=False)
```

Comment:
Add a small comment similar to the one above stating that only the endpoint is deleted.

Reply:
"endpoint endpoint", there are double "endpoint" in here.

Drop-in

Only delete the SageMaker endpoint, while keeping the corresponding endpoint configuration.